tests/i18n/tests.py

```diff
@@ -396,6 +396,17 @@ def test_template_tags_pgettext(self):
             rendered = t.render(Context())
             self.assertEqual(rendered, '2 andere Super-Ergebnisse')
 
+            # Using 'force_escape'
+            t = Template(
+                '{% load i18n %}{% blocktrans force_escape %}'
+                'Escaped content - {{ content }}{% endblocktrans %}'
+            )
+            rendered = t.render(Context({'content': '<b>"Escaped" content to try \'force_escape\''
+                                                    'argument & check for errors.</b>'}))
```

Comment:
please use hanging indent to make better use of space, e.g.
`context = Context({'content': '<b>"Escaped" content to try \'force_escape\ argument & check for errors.</b>'})`
might be good here